capstone/pom.xml

```diff
@@ -36,6 +36,33 @@
       <artifactId>spring-cloud-gcp-starter-data-datastore</artifactId>
       <version>1.2.3.RELEASE</version>
     </dependency>
+
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
+      <version>1.9.59</version>
+    </dependency>
+
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-test</artifactId>
+      <version>2.2.2.RELEASE</version>
```

Comment:
scope test

Reply:
I'm not sure what this means. Am I using the wrong dependency?

Reply:
You just need to add 
`<scope>test</scope>` so that this dependency doesn't get bundled into the jar that you deploy, since it's only used during testing.